src/components/ThorFreeFeeBanner/ThorFreeFeeBanner.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Card, CardBody } from '@chakra-ui/react'
+import { Text } from 'components/Text'
+
+import icons from './asset-icons.png'
+
+// const cardHover = {
+//   textDecoration: 'none',
+//   opacity: '.7',
+// }
```

Comment:
nitpick:

```suggestion

// const cardHover = {
//   textDecoration: 'none',
//   opacity: '.7',
// }
```

Reply:
I wanted to keep all those comments so we don't forget this stuff when the link is ready to be sent to prod, wdyt?

Reply:
Ah got it, can add a todo if it's less distracting if you want

Reply:
Added in [68096ca](https://github.com/shapeshift/web/pull/8044/commits/68096ca8f6852c390173b26034510d54e6b72296) but kept the comment so it's a no-brain great value work when implementing it (it's going to be in the next few days so! shouldn't stay as it is)